logistics_project/apps/malawi/templates/malawi/new/reports/stock-status.html

```diff
@@ -11,9 +11,6 @@
 <div class="module">
     <h2>Current stock status by product</h2>
     {% warehouse_product_availability_summary location window_date default_chart_width 300 is_facility %}
-	{% with product_table as table %}
```

Comment:
yeah i grepped the codebase and it turned up nothing. when it loaded in the page it just loaded an empty html table.